### PR TITLE
Cleanup repository

### PR DESCRIPTION
**Pull request by susnux**
* Remove unneeded handlebars scrips
* Fix stylelint config and add CI workflow for it
* Move JS mocks so they are located next to the tests
* Migrate deprecated `::v-deep` to `:deep`

### CODE REVIEW

File: src/__tests__/Activity.test.ts [hunk 1]
```diff
@@ -24,9 +24,9 @@ import { afterAll, beforeAll, expect, test, vi } from 'vitest'
 import { nextTick } from 'vue'
 
 import Activity from '../components/Activity.vue'
-import ActivityModel from '../models/ActivityModel.ts'
```
Comment (artonge) [inline src/__tests__/Activity.test.ts:27]: IIRC, we agreed to keep file extensions in imports, so please add it back :)

**Review by artonge — CHANGES_REQUESTED** [review]

File: src/__tests__/Activity.test.ts (hunk 1 above)
Comment (susnux) [reply src/__tests__/Activity.test.ts:27]: Yes, fixed it.

**Review by artonge — APPROVED** [review]

**Comment by susnux** [comment]
> Node CI is failing a lot recently. Has there been a change in node or npm version?

No there is something weird going on, the assets I compile locally slightly differ from what the CI is compiling.
But I use exactly the same NPM and node version, same dependencies (npm ci).

**Pull request merged**